### PR TITLE
Use network env var in getSpaceUri

### PR DESCRIPTION
**Pull request by mktcode**
As suggested here: https://github.com/snapshot-labs/snapshot.js/pull/420

Would allow to set `VITE_DEFAULT_NETWORK=4` and use a Rinkeby ENS domain to create a space.

### CODE REVIEW

**Review by ChaituVR — APPROVED** [review]

File: src/views/SpaceSettings.vue [hunk 1]
```diff
@@ -237,7 +237,10 @@ function formatSpace(spaceRaw) {
 watchEffect(async () => {
   if (!props.spaceLoading) {
     try {
-      const uri = await getSpaceUri(props.spaceId);
+      const uri = await getSpaceUri(
+        props.spaceId,
+        import.meta.env.VITE_DEFAULT_NETWORK
```
Comment (ChaituVR) [inline src/views/SpaceSettings.vue:242]: Maybe we can make this optional?

Comment (mktcode) [reply src/views/SpaceSettings.vue:253]: What do you mean? It is. The second parameter of `getSpaceUri` deafults to '1' same as `VITE_DEFAULT_NETWORK`. I'm just not 100% sure if `VITE_DEFAULT_NETWORK` is the best choice for this.

Comment (ChaituVR) [reply src/views/SpaceSettings.vue:253]: Oh ok, cool then :) I forgot about the default param on getSpaceUri

**Comment by midgerate** [comment]
I dont fully understand the impact of this change. If it doesn't affect anything you have tested it, I can approve it. @mktcode

**Comment by mktcode** [comment]
@midgerate See here: https://github.com/snapshot-labs/snapshot.js/pull/420
It just allows to use the new `network` param added there, so `getSpaceUri` can look up ENS domains on different networks, like Rinkeby for testing.

**Review by bonustrack — APPROVED** [review]

**Pull request merged**